src/sentry/integrations/github/integration.py

```diff
@@ -124,6 +124,18 @@ def message_from_error(self, exc):
         else:
             return ERR_INTERNAL
 
+    def has_repo_access(self, repo):
+        client = self.get_client()
+        try:
+            # make sure installation has access to this specific repo
+            # use hooks endpoint since we explicity ask for those permissions
+            # when installing the app (commits can be accessed for public repos)
+            # https://developer.github.com/v3/repos/hooks/#list-hooks
+            client.repo_hooks(repo.config['name'])
```

Comment:
this is what we're using here: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/repository.py#L25

so seems like it's a good idea to keep it consistent